recipes/default.rb

```diff
@@ -8,6 +8,7 @@
 
 # Make sure w32time service is running
 service 'w32time' do
```

Comment:
`run_as_user` is a property of the `windows_service` resource, please make foodcritic happy :)